js/angular-crmMailing/directives.js

```diff
@@ -129,43 +129,36 @@
           }
           else {
             schedule.mode = 'now';
+            schedule.datetime = '';
           }
-          validate();
         };
 
         var updateParent = (function () {
           switch (schedule.mode) {
             case 'now':
               ngModel.$setViewValue(null);
-              schedule.datetime = ' ';
+              schedule.datetime = '';
               break;
             case 'at':
+              schedule.datetime = schedule.datetime || '?';
```

Comment:
When this was ' ' it didn't work, something was forcing it back to ''. But any other character works. The unit tests pass either way, but to reproduce in the UI, try changing this character to ' ' and then click on the "At" radio - it will immediately get forced back to "Now"